linux_os/guide/system/logging/ensure_rsyslog_log_file_configuration/rsyslog_files_permissions/oval/shared.xml

```diff
@@ -6,9 +6,9 @@
         <platform>multi_platform_rhel</platform>
         <platform>multi_platform_fedora</platform>
         <platform>Debian 8</platform>
-        <platform>Ubuntu 1404</platform>
-        <platform>Ubuntu 1604</platform>
-	<platform>Ubuntu 1804</platform>
+        <platform>Ubuntu 14.04</platform>
```

Comment:
I still can grep `Ubuntu 1404` without the dot in shared/templates/template_OVAL_service_disabled and elsewhere.

Reply:
Fixed.